pkg/web/api/v1/apikey.go

```diff
@@ -27,6 +32,10 @@ type APIKeyList struct {
 	APIKeys []*APIKey  `json:"api_keys"`
 }
 
+type APIKeyQuery struct {
```

Comment:
A query has been added so that we'll know which modal should be displayed to the user.